Software/app/secure_element_lorawan/README.md

```diff
@@ -0,0 +1,32 @@
+# secure_element_lorawan
+
+## Description
+
+This application showcases LoRaWAN functionality using a HW secure element (ATECC608A-TNGLORA).
```

Comment:
maybe add a link from our docs about the secure element?

Reply:
Good idea